config/default/contact-summary-extras.js

```diff
@@ -0,0 +1,445 @@
+const moment = require('moment');
+const today = moment().startOf('day');
+
+const isReportValid = function (report) {
+  if (report.form && report.fields && report.reported_date) return true;
+  return false;
+};
+//
+
+const pregnancyForms = ['pregnancy'];
+
+const antenatalForms = ['pregnancy_home_visit'];
+
+const deliveryForms = ['delivery'];
+
+const pregnancDangerSignForms = ['pregnancy', 'pregnancy_home_visit', 'pregnancy_danger_sign', 'pregannacy_danger_sign_follow_up'];
+
+
+const MAX_DAYS_IN_PREGNANCY = 42 * 7;  // 42 weeks
+const AVG_DAYS_IN_PREGNANCY = 280;
+
+
+const ancDangerSigns = [
+  ['vaginal_bleeding', 'Vaginal bleeding'],
```

Comment:
How will this handle multiple languages?  Can we use translation keys instead?

Reply:
I need to conditionally show some of these danger signs in a single field of a condition card, separated by commas. I could not find a way to translate a bunch of values which may be different each time. I agree that handling multiple languages will be troublesome. Got any idea?

Reply:
Let's start a thread about this on http://communityhealthtoolkit.org/ or open an issue in webapp? I'm also unfamiliar with how to achieve this. We will need a feature request and somehow tag it with `cht-reference-app`.

Reply:
We have created an issue for it today on `medic/medic` here: #5969